### PR TITLE
fix header slightly covers .popover-content #3898

### LINKED ISSUE #3898: header slightly covers .popover-content
### Please describe the problem (or idea)
I read notes, and when I clicked to an arrow, I saw that header slightly covers appeared block. It happens when a note haven't image on top. Maybe this is not urgent, but this can be improved.

image

Maybe this block should be a little lower?

### Please show us where to look

Any note/question/... without main image, in example this https://publiclab.org/notes/amychan331/11-02-2018/outreachy-proposal-on-expanded-community-stats-system-by-amy-chan

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by sashadev-sky**
Fixes #3898  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by rishabhptr** [comment]
@sashadev-sky Thank you for the PR! Would you mind adding a screenshot showing how it looks now :smile: Thanks!

**Comment by sashadev-sky** [comment]
@rishabhptr thank you for the feedback! This is what it looks like on my localhost server:

screen shot 2018-11-25 at 11 28 03 am

the dropdown stretches further over the header if more "likes" are added.

Also, my PR failed the travis danger tests. Could you please advise how I can fix this?

**Comment by JonathanXu1** [comment]
Hi, your travis build seems to have an error similar to the one I'm getting: **The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi" exited with 1.**

I was wondering if you know why it's happening. If you've figured out a way to solve it I would really appreciate it!

File: .travis.yml [hunk 1]
```diff
@@ -27,5 +27,4 @@ script:
   - docker-compose exec web bash -c "CI=TRUE GENERATE_REPORT=true rails test -d"
   - echo -e '<?xml version="1.0" encoding="UTF-8"?>' > output.xml
   - tail -n +2 -q ./test/reports/TEST*.xml >> output.xml
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi'
-  
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then danger --verbose; fi'
```
Comment (rishabhptr) [inline .travis.yml]: This is unrelated to the PR, so you so could remove it. :smile:

**Comment by sashadev-sky** [comment]
@rishabhptr Thank you for the feedback! I'm still failing the build after rebasing. Any idea why? 
@JonathanXu1 Did you manage to resolve it?

**Comment by JonathanXu1** [comment]
No I never got help from mentors regarding that issue.. sorry!

**Comment by sashadev-sky** [comment]
@rishabhptr Rebasing worked, thank you! Can I have a review please? 
@JonathanXu1 updating you in case you're still curious  :+1:

**Comment by JonathanXu1** [comment]
Thanks for the heads up! It was very helpful

**Comment by jywarren** [comment]
Oh this looks lovely! Thanks! I'm just wondering about one last thing - if the message about the page not having tags is gone, and the menu button is closer up against the top menu, does it still work, or does it get cut off? Could you try resolving the alert message by adding some tag to the page, and reloading and screenshotting again? 

See how i'm not sure if this is both obscured by the header but also cut off by the page height?

image

**Comment by sashadev-sky** [comment]
@jywarren Without the alert message it still fits into the page height: 
(below is the same view from the 'moderator' and 'user' accounts, respectively)

screen shot 2018-11-29 at 10 48 16 pm

screen shot 2018-11-29 at 10 50 11 pm

**Comment by jywarren** [comment]
Great work and good documentation of the issue! Thanks a ton!
